biggus/__init__.py

```diff
@@ -485,6 +485,21 @@ class Array(object):
 
     __hash__ = None
 
+    def __str__(self):
+        nbytes = self.nbytes
+        if nbytes < (1 << 10):
+            size = '{} B'.format(nbytes)
+        elif nbytes < (1 << 20):
+            size = '{:.02f} KiB'.format(nbytes / (1 << 10))
+        elif nbytes < (1 << 30):
+            size = '{:.02f} MiB'.format(nbytes / (1 << 20))
+        elif nbytes < (1 << 40):
+            size = '{:.02f} GiB'.format(nbytes / (1 << 30))
+        else:
+            size = '{:.02f} TiB'.format(nbytes / (1 << 40))
```

Comment:
This ``size`` is more useful to me as a human than the nbytes property. Would you be happy to extract this functionality into a function which can be passed any array (biggus or any other type which has a nbytes attribute)?